akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -444,7 +444,7 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef, joinConfigCompatCh
 
   def uninitialized: Actor.Receive =
     ({
-      case InitJoin =>
+      case InitJoin(_) =>
```

Comment:
Good catch 👍

Reply:
It's an epic story of where such refactoring wouldn't go wrong if Akka Typed was used. Adding it to my list of ideas for blog posts.